test-case/check-suspend-resume.sh

```diff
@@ -132,13 +135,25 @@ main()
 # TODO: remove this after issue fixed.
     sleep 1
 
+
+    local keep_modules=true
+    if [ ${OPT_VAL['u']} = 1 ]; then
+        keep_modules=false
+    fi
+    $keep_modules || "$TOPDIR"/tools/kmod/sof_remove.sh ||
+        die "Failed to unload audio drivers"
+
     expected_wakeup_count=$(cat /sys/power/wakeup_count)
     expected_stats_success=$(cat /sys/power/suspend_stats/success)
     save_initial_stats
     for i in $(seq 1 $loop_count)
     do
         sleep_once "$i"
     done
+
+    $keep_modules || "$TOPDIR"/tools/kmod/sof_remove.sh ||
```

Comment:
should this be sof_insert? I am not sure why we use sof_remove twice, unless it's a copy-paste issue

Reply:
Good catch, thank you! Late Friday code...

Fixed. Also added logic not to do anything when the drivers are already unloaded.